examples/s3-object-lambda/src/s3-object-lambda-stack.ts

```diff
@@ -70,51 +62,14 @@ export class S3ObjectLambdaStack extends pulumicdk.Stack {
             sourceAccount: cdk.Aws.ACCOUNT_ID,
         });
 
-        // Associate Bucket's access point with lambda get access
-        const policyDoc = new iam.PolicyDocument();
-        const policyStatement = new iam.PolicyStatement({
-            effect: iam.Effect.ALLOW,
-            actions: ['s3:GetObject'],
-            principals: [
-                new iam.ArnPrincipal(this.asOutput(retrieveTransformedObjectLambda.role?.roleArn) as unknown as string),
-            ],
-            resources: [this.asOutput(`${accessPoint}/object/*`) as unknown as string],
-        });
-        policyStatement.sid = 'AllowLambdaToUseAccessPoint';
-        policyDoc.addStatements(policyStatement);
-
-        const ap = new aws.s3.AccessPoint('exampleBucketAP', {
-            // CDK property can be passed to a Pulumi resource
-            bucket: this.asOutput(bucket.bucketName),
-            name: S3_ACCESS_POINT_NAME,
-            policy: policyDoc.toJSON(),
-        });
-
-        // Access point to receive GET request and use lambda to process objects
-        const objectLambdaAP = new s3ObjectLambda.CfnAccessPoint(this, 's3ObjectLambdaAP', {
-            name: OBJECT_LAMBDA_ACCESS_POINT_NAME,
-            objectLambdaConfiguration: {
-                // a pulumi resource property can be passed to a cdk resource
-                supportingAccessPoint: pulumicdk.asString(ap.arn),
-                transformationConfigurations: [
-                    {
-                        actions: ['GetObject'],
-                        contentTransformation: {
-                            AwsLambda: {
-                                FunctionArn: `${retrieveTransformedObjectLambda.functionArn}`,
-                            },
-                        },
-                    },
-                ],
-            },
+        const objectLambdaAP = new s3ObjectLambda.AccessPoint(this, 's3-object-lambda-ap', {
```

Comment:
This was an old test so don't know if it was using L1s due to the alpha module not existing yet. Updating this test to use the L2s instead.